### PR TITLE
wechat: Add version 3.7.5.23

### LINKED ISSUE #7160: (Request) add Wechat
PackageIdentifier: Wechat
PackageVersion: 3.3
PackageLocale: en-US
Publisher: Tencent
PackageName: wechat
License: proprietary
ShortDescription: Chinese messaging app
Installers:

Architecture: x64
InstallerType: exe
InstallerUrl: https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe
ManifestType: singleton
ManifestVersion: 1.0.0

### PR DESCRIPTION
**Pull request by b4imetu**
Closes #7160 

code modified from [kidonng/sushi](https://github.com/kidonng/sushi/blob/master/bucket/wechat.json) 

**p.s.** *WeChat* is the **most popular instant messenger** app in China.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/wechat.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
+            "Remove-Item \"$dir\\`$_15_\" -Force -Recurse",
+            "Remove-Item \"$dir\\`$R5\" -Force -Recurse"
+        ]
+    },
+    "shortcuts": [
+        [
+            "wechat.exe",
+            "WeChat 微信"
+        ]
```
Comment (issaclin32) [inline bucket/wechat.json:23]:
```suggestion
        [
            "wechat.exe",
            "WeChat"
        ]
```

File: bucket/wechat.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "notes": "We don't persist your WeChat data storaged in \"$env:APPDATA\\Tencent\\WeChat\".",
+    "url": "https://webcdn.m.qq.com/spcmgr/download/WeChatSetup_3.7.5.23.exe#/dl.7z",
+    "hash": "c91856d0721b09a10c7b37013ca93cabb5e618ea768e66a70141052930c0c351",
+    "installer": {
+        "script": [
+            "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
+            "Remove-Item \"$dir\\`$_15_\" -Force -Recurse",
+            "Remove-Item \"$dir\\`$R5\" -Force -Recurse"
+        ]
```
Comment (issaclin32) [inline bucket/wechat.json:17]:
```suggestion
        "script": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\`$_15_\", \"$dir\\`$R5\" -Force -Recurse",
```

File: bucket/wechat.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "3.7.5.23",
+    "description": "Free messaging and calling app by Tencent",
+    "homepage": "https://pc.weixin.qq.com/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.wechat.com/en/service_terms.html"
+    },
+    "notes": "We don't persist your WeChat data storaged in \"$env:APPDATA\\Tencent\\WeChat\".",
```
Comment (issaclin32) [inline bucket/wechat.json:9]:
```suggestion
```
I don't think this is needed. Many apps in *Scoop* stores their data in `$Env:AppData`, but we don't show it explicitly.

**Comment by issaclin32** [comment]
Glad to see this app being added to *Scoop*, but I got a small problem.
The UI is in **Traditional Chinese** even if I
**(1)** Download the [English version](https://pc.weixin.qq.com/?t=win_weixin&lang=en_US)
**(2)** Switch Win10 system language to English.

I guess this depends on **"what language is the system initially installed"**. (Otherwise it should be **Simplified Chinese** by default)

So we probably need someone using a (natively) English version of Win10 to test it out.
wechat

**Comment by b4imetu** [comment]
@issaclin32 
I use en_US as my default installation language for Windows 11, and my Windows version is 22621.232.
WeChat tested fine with me.
I think what language WeChat displays by default depends on `Control Panel > Region > System Locale`. Maybe you can try it. Remember to restart your system after changing the settings.

**Pull request merged**